spec/lib/dotiw_spec.rb

```diff
@@ -339,7 +339,37 @@
       [START_TIME,
        START_TIME + 1.days,
        { highest_measures: 1, only: %i[years months] },
-       'less than 1 month']
+       'less than 1 month'],
+      [START_TIME,
+       START_TIME + 1.days,
+       { highest_measures: {} },
+       '1 day'],
+      [START_TIME,
+       START_TIME + 1.year + 1.minute,
+       { highest_measures: { remainder: :ceiling } },
+       '2 years'],
+      [START_TIME,
+       START_TIME + 1.day + 2.hours + 30.minutes,
+       { highest_measures: { max: 2, remainder: :round } },
+       '1 day and 3 hours'],
+      [START_TIME,
+       START_TIME + 1.day + 23.hours + 59.minutes + 59.seconds,
+       { highest_measures: { max: 3, remainder: :round } },
+       '2 days'],
+      [START_TIME,
+       START_TIME + 1.day,
+       { highest_measures: { remainder: :ceiling }, only: :months },
+       'less than 1 month'],
+      [START_TIME,
+       # Simplistic rounding: one would expect this to round up to a second week.
+       START_TIME + 1.week + 3.days + 23.hours,
+       { highest_measures: { remainder: :round } },
+       '1 week'],
+      [START_TIME,
+       # Simplistic rounding: in some months, 15 days is less than half, but we always round it up.
+       START_TIME + 1.month + 14.days,
+       { highest_measures: { remainder: :round } },
+       '2 months'],
```

Comment:
Add an example with an explicit `:floor`. 

Add a test where the `remainder` value is invalid, expect an exception.